### DISCUSSION TITLE
Is is possible to get the k value from the data used to plot the elbow graph?

### DISCUSSION THREAD

**Discussion by batmanscode** [pycaret/pycaret#1538] [General]
This is about the elbow graph in the clustering module. Is there a way to get the raw data that's used to make the plot?

![image](https://user-images.githubusercontent.com/29989939/131021922-f63f8a97-f8f3-4475-b08d-cd35d60206ad.png)

It will be useful to: create model > get k from the plot data > update/finetune model

**Comment by usyed15181** [comment]
Is it possible to get the elbow value? In the example above, can we pull 4 because that's the elbow.

**Reply by moezali1** [reply]
Not possible yet.

**Reply by batmanscode** [reply]
> Is it possible to get the elbow value? In the example above, can we pull 4 because that's the elbow.

I used the silhouette coefficient instead of the elbow value as a work around
